monai/transforms/post/dictionary.py

```
@@ -508,6 +513,206 @@ def __init__(self, keys: KeysCollection, output_key: str | None = None, num_clas
+    def _determine_shape(
+        self, points: Any, static_shape: tuple[int, ...] | None, data: Mapping[Hashable, Any], ref_key: Hashable | None
+    ) -> tuple[int, ...]:
+        points_t = convert_to_tensor(points, dtype=torch.float32, track_meta=False)
+        if points_t.ndim != 2:
+            raise ValueError(f"{self._ERR_INVALID_POINTS} Got {points_t.ndim}D tensor.")
+        spatial_dims = int(points_t.shape[-1])
+        if static_shape is not None:
+            if len(static_shape) != spatial_dims:
+                raise ValueError(
+                    f"Provided static spatial_shape has {len(static_shape)} dims; expected {spatial_dims}."
+                )
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)